src/main/java/de/blau/android/dialogs/Layers.java

```diff
@@ -701,6 +702,7 @@ private AlertDialog buildImagerySelectDialog(@Nullable final TableRow row, @Null
 
         final ImageryListAdapter adapter = new ImageryListAdapter(ids, currentId, isOverlay, buttonLayoutParams,
                 new LayerOnCheckedChangeListener(activity, dialog, row, layer, ids));
+        adapter.addItemClickListener(this);
```

Comment:
While a bit a matter of taste, I would have simply provided the listener as a lambda here, so something like

```
      adapter.addItemClickListener((String id) -> {
            TileLayerSource l = TileLayerSource.get(getContext(), id, true);
            if (l != null) {
                LayerInfo f = new ImageryLayerInfo();
                f.setShowsDialog(true);
                Bundle args = new Bundle();
                args.putSerializable(ImageryLayerInfo.LAYER_KEY, l);
                f.setArguments(args);
                LayerInfo.showDialog(getActivity(), f);
            }
        });
```